frontstage/controllers/collection_instrument_controller.py

```diff
@@ -113,16 +113,17 @@ def upload_collection_instrument(file, case: dict, business_party: dict, party_i
             gcp_survey_response.upload_seft_survey_response(case, file_contents, file_name, survey_ref)
             ci_post_case_event(case_id, party_id, "SUCCESSFUL_RESPONSE_UPLOAD")
             logger.info("Successfully uploaded collection instrument", case_id=case_id, party_id=party_id)
+            return None
         except FileTooSmallError:
             ci_post_case_event(case_id, party_id, "UNSUCCESSFUL_RESPONSE_UPLOAD")
-            raise CiUploadError(FILE_TOO_SMALL)
+            return FILE_TOO_SMALL
```

Comment:
Just a question, can a file ever bee too small for the purposes of the SEFT upload?

Reply:
We have a method that checks if a file is above a byte but i don't think a xlsx could ever be less than a byte.

```
    def check_if_file_size_too_small(file_size) -> bool:
        return file_size < 1
```